src/agoradatatools/etl/transform/model_details.py

```diff
@@ -206,10 +215,33 @@ def transform_model_details(
             "alzforum_id": model_row["alzforum_id"],
             "genotype": model_row["genotype"],
             "aliases": model_row["aliases"],
+            "gene_expression": None,
+            "disease_correlation": None,
+            "allen_institute": None,
             "genetic_info": genetic_info,
             "biomarkers": model_biomarkers,
             "pathology": model_pathology,
         }
+
+        # Add gene expression and disease correlation links if they exist
+        if model_name in model_results_info_df["name"].values:
+            model_results_info_row_dict = model_results_info_df[
+                model_results_info_df["name"] == model_name
+            ].to_dict("records")[0]
+
+            model_entry["gene_expression"] = (
+                f"comparison/expression?model={model_name}"
+                if pd.notna(model_results_info_row_dict["gene_expression"])
+                and bool(model_results_info_row_dict["gene_expression"])
+                else None
+            )
+            model_entry["disease_correlation"] = (
+                f"comparison/correlation?model={model_name}"
+                if pd.notna(model_results_info_row_dict["disease_correlation"])
+                and bool(model_results_info_row_dict["disease_correlation"])
+                else None
+            )
```

Comment:
Is there any code to fill out the `allen_instutute` (`spatial_transcriptomics`) field?

Reply:
There is not; I was told that the data would be hosted for visualization on the Allen Institute site at one point, but that hasn't materialized. AFAIK the spatial data hasn't been released anywhere at all yet, but will be going into the ADKP in time. Once it lives somewhere, as data or as visualizations, we can fill in the blank.

Reply:
Got it! Thanks.